site/app/templates/notebook/Notebook.twig

```diff
@@ -61,6 +61,7 @@
                 <div
                     id="codebox_{{ num_codeboxes }}"
                     name="codebox_{{ num_codeboxes }}"
+                    class="{{ cell.filename }}"
```

Comment:
The filename probably shouldn't be stored in the class because we would probably never need to style these. Also, if we tried, we would have to escape the period in the filename which would be a mess. I recommend storing it in a data attribute.
```suggestion
                    data-filename="{{ cell.filename }}"
```